prime-router/metadata/hl7_mapping/datatypes/HumanName/XPN.yml

```diff
@@ -0,0 +1,77 @@
+elements:
+
+  - name: xpn-family-name
+    value: [ '%resource.family' ]
+    hl7Spec: [ '%{hl7xpnField}-1-1' ]
+
+  - name: xpn-humanname-own-prefix
+    value: [ '%resource.family.extension(%`ext-humanname-own-prefix`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-2' ]
+
+  - name: xpn-humanname-own-name
+    value: [ '%resource.family.extension(%`ext-humanname-own-name`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-3' ]
+
+  - name: xpn-humanname-partner-prefix
+    value: [ '%resource.family.extension(%`ext-humanname-partner-prefix`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-4' ]
+
+  - name: xpn-humanname-partner-name
+    value: [ '%resource.family.extension(%`ext-humanname-partner-name`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-1-5' ]
+
+  - name: xpn-given-name
+    value: [ '%resource.extension(%`rsext-xpn2-given-name`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-2' ]
+
+  - name: xpn-second-given-name
+    value: [ '%resource.extension(%`rsext-xpn3-second-given-name`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-3' ]
+
+  - name: xpn-suffix
+    value: [ '%resource.extension(%`rsext-xpn4-suffix`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-4' ]
+
+  - name: xpn-prefix
+    value: [ '%resource.prefix' ]
+    hl7Spec: [ '%{hl7xpnField}-5' ]
+
+  - name: xpn-degree
+    value: [ '%resource.extension(%`rsext-xpn6-degree`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-6' ]
+
+  - name: xpn-name-type-code
+    value: [ '%resource.extension(%`rsext-xpn7-name-type-code`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-7' ]
+
+  - name: xpn-name-representation-code
+    value: [ '%resource.extension(%`rsext-xpn8-name-representation-code`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-8' ]
+
+  - name: xpn-name-context
+    value: [ '%resource.extension(%`rsext-xpn9-name-context`).value' ]
+    hl7Spec: [ '%{hl7xpnField}-9' ]
+
+  - name: xpn-name-validity-range-start
+    value: [ '%resource.extension(%`rsext-xpn10-name-validity-range`).value.start' ]
+    hl7Spec: [ '%{hl7xpnField}-10-1' ]
+
+  - name: xpn-name-validity-range-end
+    value: [ '%resource.extension(%`rsext-xpn10-name-validity-range`).value.end' ]
+    hl7Spec: [ '%{hl7xpnField}-10-2' ]
```

Comment:
(informational) This is another place where the impending DTM mapping could warrant changes, whether it's part of this PR or not.